pkg/reconciler/revision/resources/deploy.go

```diff
@@ -136,49 +135,60 @@ func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig
 	return podSpec, nil
 }
 
-// BuildUserContainer makes a container from the Revision template.
-func BuildUserContainer(rev *v1.Revision) *corev1.Container {
-	userContainer := rev.Spec.GetContainer().DeepCopy()
```

Comment:
previously we did a DeepCopy before returning, now we directly modify the containers in spec. Is this safe?

Reply:
True
But when i removed it works without any issues :thinking: 
I think it should better to keep DeepCopy 
Reverted back to use DeepCopy again